src/mpl/include/mpl_base.h

```diff
@@ -159,5 +159,8 @@ typedef bool _Bool;
 
 #define MPL_ROUND_UP_ALIGN(a, alignment) (((a) + ((alignment) - 1)) & (~((alignment) - 1)))
 #define MPL_ROUND_DOWN_ALIGN(a, alignment) ((a) & (~((alignment) - 1)))
+#define MPL_ROUNDUP(alignment) \
```

Comment:
Oh, we already have `MPL_ROUND_UP_ALIGN`, just use it, there is no need for an extra macro.

Reply:
Done. A small note: the existing macro assumes you are aligning to a power of 2, while the new one does not. I do not think it is an issue for this use note, but the semantics of the two are not identical